data-sources/datazoom/config.yml

```diff
@@ -1,7 +1,7 @@
 id: datazoom
 displayName: Datazoom
 description: |
-  The Datazoom quickstart provides a fast and easy launching point into video metrics built on top of New Relic One dashboards, allowing you to gain insights about your viewers, content, ads and your platform's performance. Customize the dashboard to use only the metrics that fit your goals and easily edit metric calculations and tailor them to your needs.
+  The Datazoom quickstart provides a fast and easy launching point into video metrics built on top of New Relic One dashboards, allowing you to gain insights about your viewers, content, ads and your platform's performance.
```

Comment:
this might be a producty question, so unsure if it applies but would we want to remove/replace references to "quickstart" since these are data sources?

Reply:
yeah I figure I'll leave them for now, I imagine there's a lot of that :')